app/server/devServerMain.ts

```diff
@@ -118,17 +119,18 @@ export async function main() {
   log.info("== staticServer");
   const staticPort = getPort("STATIC_PORT", 9001);
   process.env.APP_STATIC_URL = `http://localhost:${staticPort}`;
-  await mergedServerMain(staticPort, ["static"]);
+  await MergedServer.create(staticPort, ["static"]).then((s) => s.run());
```

Comment:
weak preference for spelling this out with an intermediate variable rather than using a `then`